src/firmware/app/control/trajectory_planner.h

```diff
@@ -26,28 +30,37 @@ typedef struct FirmwareRobotPathParameters
     //  point along the trajectory. This factor limits the maximum delta-velocity
     // and
     //  also the max speed around curves due to centripetal acceleration [m/s^2]
-    float max_allowable_acceleration;
-    // The maximum speed allowable at any point along the trajectory```
-    float max_allowable_speed;
+    float max_allowable_linear_acceleration;
+    // The maximum allowable angular acceleration allowable at any point on the
+    // orientation profile The maximum speed allowable at any point along the trajectory
+    float max_allowable_linear_speed;
```

Comment:
comment about acceleration above a variable for speed

Reply:
Good catch, fixed 👍

Reply:
I don't see the change

Reply:
Actually fixed this time 👍